lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -910,6 +911,39 @@ export class UserAgentApplication {
     return adalFrame;
   }
 
+  private pollWindowForHash(contentWindow: Window): Promise<string> {
+      return new Promise((resolve, reject) => {
```

Comment:
nit: [consistency] You can also use async await instead of `new Promise`.

Reply:
I can't actually, since the promise will be resolved inside the `setInterval`.